frontend/src/components/layout-content/index.vue

```diff
@@ -21,36 +21,32 @@
-                                            <slot name="leftToolBar" v-if="slots.leftToolBar"></slot>
-                                        </template>
-                                    </back-button>
-                                </div>
-                                <div class="flex flex-wrap gap-3">
-                                    <slot name="rightToolBar" v-if="slots.rightToolBar"></slot>
-                                </div>
+                        <div v-if="showBack" class="flex flex-wrap gap-4 sm:justify-between">
+                            <back-button
+                                :path="backPath"
+                                :name="backName"
+                                :to="backTo"
+                                :header="title"
+                                :reload="reload"
+                            >
+                                <template v-if="slots.leftToolBar" #buttons>
+                                    <div class="flex flex-wrap gap-2 items-center justify-start">
+                                        <slot name="leftToolBar" v-if="slots.leftToolBar"></slot>
+                                    </div>
+                                </template>
+                            </back-button>
+                            <div class="flex flex-wrap gap-3">
+                                <slot name="rightToolBar" v-if="slots.rightToolBar"></slot>
                             </div>
                         </div>
-                        <div v-else>
-                            <div class="flex flex-wrap gap-4 sm:justify-between">
-                                <div class="flex gap-2 flex-wrap items-center justify-start">
-                                    <slot name="leftToolBar" v-if="slots.leftToolBar"></slot>
-                                    <slot name="buttons" v-if="slots.buttons"></slot>
-                                </div>
-                                <div class="flex flex-wrap gap-3" v-if="slots.rightToolBar || slots.rightButton">
-                                    <slot name="rightToolBar"></slot>
-                                    <slot name="rightButton"></slot>
-                                </div>
+                        <div v-else class="flex flex-wrap gap-4 sm:justify-between">
+                            <div class="flex gap-2 flex-wrap items-center justify-start">
+                                <slot name="leftToolBar" v-if="slots.leftToolBar"></slot>
+                                <slot name="buttons" v-if="slots.buttons"></slot>
+                            </div>
+                            <div class="flex flex-wrap gap-3" v-if="slots.rightToolBar || slots.rightButton">
+                                <slot name="rightToolBar"></slot>
+                                <slot name="rightButton"></slot>
                             </div>
                         </div>
 
```

Comment:
The major changes between the two versions you provided occur at line `21`, which is causing an inconsistency in styling. Specifically, the `<div>` element inside a card has both rounded corners (using CSS for box-shadow) and borders around it using display flex with border-radius set to 4px instead of being specified directly on that same div.

To correct these discrepancies:

- Set the CSS properties (`box-shadow` and other relevant styles) within `.content-container__title`.
- Remove unnecessary margin property from `.content-container__title`. The content containers should be responsive if needed, not always have margins.
- Ensure all instances of `.content-container .btn-primary.btn-danger` are correctly aligned without padding.
- Update back-to-front styling consistently across various components to maintain a consistent layout structure throughout this module.

These adjustments will help improve the coherence and look of the overall design.